proprietary/tokens/README.md

```diff
@@ -1,3 +1,229 @@
+Import it after the main stylesheet for the correct result.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
+import "@amsterdam/design-system-tokens/dist/compact.css";
+```
+
+### Three layers
+
+The tokens are organised in three layers: brand, common and component tokens.
+
+#### Brand tokens
+
+These express the corporate identity of the City of Amsterdam.
+They are our fundamental selection from all possible colours, text characteristics, spacing lengths, border widths, etc.
+Examples:
+
+```css
+:root {
+  --ams-color-primary-red: #ec0000;
+  --ams-space-md: 1rem;
+  --ams-proportion-wide: 4/3;
+  --ams-border-width-lg: 0.1875rem;
+}
+```
+
+Brand tokens must only be referenced by common tokens; not by component tokens or component stylesheets.
+(We are currently working to apply this rule to our own libraries.)
+
+Find the [list of brand tokens](https://github.com/Amsterdam/design-system/tree/main/proprietary/tokens/src/brand/ams) on GitHub.
+
+#### Common tokens
+
+Related components share visual design characteristics.
+E.g. all kinds of links have the same colour, and the borders of various form inputs are equally thick.
+Common tokens express these relations and streamline future changes.
+
+Design system components use common tokens where possible.
+The same goes for custom components that you may create in your application.
+
+```html
+<input class="my-input" type="text" />
+```
+
+```css
+.my-input {
+  font-size: var(--ams-common-input-font-size);
+  border-width: var(--ams-common-input-border-width);
+}
+```
+
+(This is an example; these specific tokens do not exist yet.)
+
+Find the [list of common tokens](https://github.com/Amsterdam/design-system/tree/main/proprietary/tokens/src/common/ams) on GitHub.
+
+#### Component tokens
+
+Every design system component defines a token for every property that expresses branding and uses it in its stylesheet.
+
+Use these tokens when recreating an existing component to receive the correct values for them – now and in the future.
+Do not apply these tokens to other components – components must be independent.
+
+```html
+<button class="my-button" type="button">Button label</button>
+```
+
+```css
+.my-button {
+  font-family: var(--ams-button-font-family);
+  background-color: var(--ams-button-primary-background-color);
+}
+```
+
+Find the [list of component tokens](https://github.com/Amsterdam/design-system/tree/main/proprietary/tokens/src/component/ams) on GitHub.
+
+### Overriding tokens
+
+This package allows creating a theme to reuse our components for a different brand.
+This is a key feature of [NL Design System](https://nldesignsystem.nl/) of which we are part.
+
+However, websites and applications for the City of Amsterdam must follow the design system as closely as possible.
+We repeat: websites and applications for the City of Amsterdam must follow the design system as closely as possible.
```

Comment:
This is kind of funny 😁 For that reason alone, maybe we should keep it. Generally, you would use `em` or `strong` for this though.